deploy/manifests/sti-dev-v2/us-east-2/cluster/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base/ingress-nginx
+  - kube-system
+  - cert-manager
+  - external-dns
+  - storetheindex
+  - cluster-autoscaler
+  - monitoring
+  - aws-ebs-csi-driver
+  - promtail
```

Comment:
Ditto re moving or using some centralised place to set up common controllers; i was hoping at the very least we centrally configured controllers/mechanisms for monitoring, log forwarding, CSI drivers, autoscaler, dns management, certificate management. 

Could you help me get my head around what the plan is going forward? Is the expectation that every cluster installs all these controllers themselves?

Reply:
Yes, this will be moved out as a part of storetheindex v3 into seperate profiles. We want to make this iterative. So this will be removed in the next few weeks